cmd/configuration/configuration.go

```diff
@@ -69,6 +71,10 @@ func configInitFunc(ctx context.Context, args []string, cmdCtx cmdcore.CommandCo
 
 func initFlytectlConfig(ctx context.Context, reader io.Reader) error {
 
+	if err := util.SetupFlyteDir(); err != nil {
```

Comment:
Lets also add a message on where the config is initialized so that the user can check it